packit_service/worker/helpers/testing_farm.py

```diff
@@ -523,6 +556,16 @@ def _payload(
         env_variables = self.job_config.env if hasattr(self.job_config, "env") else {}
         predefined_environment.update(env_variables)
 
+        # User-defined variables from comments have priority
+        if self.is_comment_event:
+            if self.comment_arguments.envs is not None:
+                env_variables = {
+                    k: v
+                    for k, v in self.comment_arguments.envs.items()
+                    if v is not None
+                }
```

Comment:
what about a use-case of "unsetting" env var via comment arg?

Reply:
I didn't think about this one. You mean something like `--env MY_ENV_VAR=""` will unset it or do you have something else how should look like? It shouldn't be a problem to implement it.

Reply:
I did some changes, please take a look :)

Reply:
yes, practically both, like if someone passes `--env ENV_VAR_TO_UNSET=  --env OTHER_ENV_VAR=value`, which would lead to the value being None